distribution/lib/Standard/Base/0.0.0-dev/src/Warning.enso

```diff
@@ -36,7 +36,7 @@ type Warning
          nature is preserved.
     reassignments : Vector.Vector Stack_Trace_Element
     reassignments self =
-        Vector.Vector_Data self.get_reassignments . map r->
+        Vector.from_array self.get_reassignments . map r->
```

Comment:
You probably want to replace `Vector_Data` with `from_polyglot_array` every time, to avoid introducing a copy. Also another argument for why `from_polyglot_array` is the wrong name for this primitive.

Reply:
This comment has many places where it should be applied. I.e. it should be applied every time `Vector_Data` is replaced in this PR.

Reply:
> You probably want to replace Vector_Data with from_polyglot_array every time, to avoid introducing a copy

Done it ce6aa5cddb9e6a. I guess I leave the rename of the `from_polyglot_array` for later... designing the right name/arguments may take a while and we already have `from_polyglot_array` to use.